app/assets/stylesheets/components/_accordion.scss

```diff
@@ -0,0 +1,109 @@
+.app-c-accordion {
+  &.js-accordion-with-descriptions {
+    margin-bottom: $gutter;
+    border-bottom: 1px solid $border-colour;
+  }
+}
+
+.app-c-accordion__header {
+  position: relative;
+  padding-top: 14px;
+  padding-bottom: 12px;
+
+  .js-accordion-with-descriptions & {
+    border-top: 1px solid $border-colour;
+    cursor: pointer;
+
+    &:hover {
+      background: $grey-3;
+    }
+  }
+}
+
+.app-c-accordion__icon {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  height: 16px;
+  width: 16px;
+  margin-top: -8px;
+  background-image: image-url('icons-plus-minus.png'); // PNG fallback for SVG
+  background: image-url("icons-plus-minus.svg"), linear-gradient(transparent, transparent); // SVG sprite
```

Comment:
What's the `linear-gradient(transparent, transparent)` bit doing here?

Reply:
That's an excellent question. My rubbish answer is that it was there already. I wonder if it's some kind of browser specific thing. Perhaps @alecgibson might have an idea?

Reply:
I think my equally rubbish answer is that it was there even when I got here.

Reply:
http://pauginer.com/post/36614680636/invisible-gradient-technique

Reply:
Full disclosure: I had to Google it (and it could definitely do with a comment!)